subcommittees/mentoring/PracticumProposal.md

```diff
@@ -0,0 +1,192 @@
+
+### Mini-lessons
+
+Each trainee chooses a topic relevant to Software and Data Carpentry (can be one of our lessons or
+their own provided their mentor approves) and teaches a 1 hour lesson at their local institution to an
+audience of their choice.  This lesson should be at least partially observed by a mentor and feedback
+provided.  Whenever possible, the mentor observing the mini-lesson should be the mentor of the previous
+mentoring element.  
+
+### General comments
+
+These two elements need not happen consecutively, but in the first implementation of this practicum
+they will be consecutive, not parallel (see timeline).  
+
+These two elements will be required to become a fully badged Software or Data Carpentry Instructor.  
+
+## Implementation Logistics
+
+### Mentoring Groups
+
+For the first practicum, all recent participants in instructor training (online 8 week, December 2 day)
+will be invited to join.  This program may be extended to include all newly trained instructors after
+we have figured out how it works.
+
+Candidates will sign up via a form that requests the following information: 
+
+- Wants to teach Software Carpentry and/or Data Carpentry 
+- Whether they want to do Python or R  
+- group member preference from their training (if any) 
+- time zone + geographic location (nearest airport), preferred time of day for meetings (daytime/evening) 
+- Language preferences (if any). 
+- When they went through instructor training 
+ 
+Based on this data, the mentoring committee will form groups of 3-4 people, prioritizing in the order
+listed above.  
+
+### Mentors
+
+An advertisement for mentors will be posted on the Software Carpentry blog, outlining the time
+commitment, what is expected, and what will be provided: 
+
+- Requirements: have taught at least 3 workshops, one of which must have been in this year.  
+- Desired skills: An interest in helping people learn, strong communication skills.  Mentors need not
+be experts in the lessons, although they should be somewhat familiar with lesson development and
+history.  Expert help will be provided to support the mentors. 
+- Time commitment: 1 hr of training (discussed below), 1 hr online / week for 3-4 weeks of mentoring,
+2-3 hrs / week on email mentoring, ~2 hrs / group member on mini-lesson preparation, observation, and
+follow-up.  Mentoring for this practicum is a time-limited focused committment; volunteering now
+doesn’t mean you’re committing to doing it again 
+- Training, discussion questions, and feedback tools will be provided 
+  
+Mentors can apply via a form that requests the following information: 
+
+- SCF experience 
+- Other teaching experience 
+- R or python preference 
+- Time zone plus preferred time of day for meetings (daytime/evening) 
+
+Mentors will be selected and matched to groups by the mentoring committee.  
+
+Before starting to meet with their groups, all mentors are required to attend the mentoring training
```

Comment:
Whether this is one hour, 30 mins or 1.5 hours should be left to the person figuring out the content in this, hence the slight fuzzyness. We are also likely to have to do some adjustments due to timezones, yes.